modules/tensor_mechanics/test/tests/material_limit_time_step/creep/tests

```diff
@@ -17,5 +20,7 @@
     override_columns = '_dt'
     override_rel_err = '1e-5'
     override_abs_zero = '1e-8'
+    issues = '#10382'
+    requirement = 'The system shall compute a non-variable value for the creep limited timestep on the initial evaluation.'
```

Comment:
I think this would be better stated as:
'The system shall not impose a limit on the time step during the initial evaluation when the creep limiting time step option is used'